src/Wannabe Edgerunner/r6/scripts/WannabeEdgerunner/track/LocKeyChecker.reds

```diff
@@ -2,19 +2,46 @@ import Edgerunning.System.EdgerunningSystem
 import Edgerunning.Common.E
 
 public class EdgerunnerInteractionChecker extends ScriptableEnv {
-  let sleep: String = "LocKey#46418";
+  let sleep: array<String>;
   let shower: String = "LocKey#46419";
   let donate: String = "LocKey#45546";
   let pet: array<String>;
   let apartment: array<String>;
+  let lover: array<String>;
+  let social: array<String>;
 
   private cb func OnLoad() {
+    this.sleep = [
+	  "LocKey#46047",  // Interacting with lover while in bed, or as part of the sleep action when they are over.
+	  "LocKey#46418"   // bed
+	];
     this.pet = [ 
       "LocKey#39016", // Iguana
       "LocKey#32348", // Cat
       "LocKey#32183", // Nibbles
-      "LocKey#32185"  // Nibbles
+      "LocKey#32185", // Nibbles
+	  "LocKey#80033", // Cat, near Pacifica Rollercoaster
+	  "Elmo", // Deceptious Cat Petting Mod Support
```

Comment:
I guess that Deceptious mod is not localized?

Reply:
Yeah, it does not look like it. I will doublecheck.

Reply:
Yup I was right. I will update this if someone creates a translation for their mod or it us updated to work differently.